packages/dashboard/src/components/SubwayNav/SubwayNavPage.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { ExampleCard, IComponentDemoPageProps, ComponentPage, PropertiesTableSet } from '@uifabric/example-app-base';
+import {
+  SubwayNavBasicExample,
+  SubwayNavWizardCompleteExample,
+  SubwayNavBasicDifferentStatesExample,
+  SubwayNavDisabledStepsExample,
+  SubwayNavSubStepsExample
+} from '@uifabric/dashboard/lib/components/SubwayNav/examples';
```

Comment:
This seems to be example one should this be inside the examples folder?

Reply:
The pattern in other components I saw was this is in the main folder.